src/components/text/Link.jsx

```diff
@@ -18,7 +18,8 @@ const SIZE = {
   OBSCURE: 'obscure'
 };
 
-const Link = ({children, href = '#', color, size = SIZE.NORMAL, unstyled, underlined, emphasised, disabled}) => {
+const Link = ({children, href = '#', color, size = SIZE.NORMAL, unstyled, underlined, emphasised, disabled,
```

Comment:
Maybe we should refactor it in the way that if link does not have href set then its disabled?